### DISCUSSION TITLE
Use github wiki pages for wiki - PR

### DISCUSSION THREAD

**Discussion by calleum** [inkytonik/kiama#31] [Ideas]
I have converted the wiki directory to be hosted on github wikis. 

Github doesn't allow PR on repo wikis, so I have added this discussion.

I did this for the convenience of having the side panel while I have been reading through documentation.
The content is available at https://github.com/cpecq/kiama/wiki - the only changes to the docs found in the wiki directory is the result of running sed to strip '.md' from references in the docs, to allow it to work within github wikis.
Thanks!

**Comment by inkytonik** [comment]
Thanks for this! I probably won't have time for a proper look until the weekend, but it sounds like a good idea to switch over.

**Comment by inkytonik** [comment]
Thanks again for the suggestion. In the end I went for the GH pages route, you can find the result [here](https://inkytonik.github.io/kiama/). Please let me know if you see any issues.
